pages/linux/resize2fs.md

```diff
@@ -0,0 +1,17 @@
+# resize2fs
+
+> Resize an ext2, ext3 or ext4 file system.
+> Does not resize the underlying partition.
+> File system must be unmounted.
+
+- Automatically resize file system:
```

Comment:
Perhaps this will read better?
```suggestion
- Automatically resize a filesystem:
```

Also, `file system` → `filesystem` I think, based on other pages (e.g. `mkfs`).

Reply:
File system seems to be the most common usage:
https://books.google.com/ngrams/graph?content=filesystem%2Cfile+system&corpus=26&year_start=1940&year_end=2019&smoothing=3&direct_url=t1%3B%2Cfilesystem%3B%2Cc0%3B.t1%3B%2Cfile%20system%3B%2Cc0

Reply:
Although manpages seem to prefer filesystem, and this repo has more cases of filesystem over file system.